shared/notifications/index.js

```diff
@@ -135,6 +142,11 @@ export const UI_NOTIFICATIONS = {
     id: 24,
     date: null,
   },
+  // This syntax is unusual, but very helpful here.  It's equivalent to `UI_NOTIFICATIONS[NOTIFICATION_DROP_LEDGER_FIREFOX] =`
+  [NOTIFICATION_DROP_LEDGER_FIREFOX]: {
+    id: Number(NOTIFICATION_DROP_LEDGER_FIREFOX),
+    date: null,
+  },
```

Comment:
@montelaidev you want to add the notification number in here?

Reply:
As a comment

Reply:
Yeah I think I was also confused about where he meant to add it.  Adding it here would increase the merge conflict work.